app/serializers/transcription_serializer.rb

```diff
@@ -1,6 +1,9 @@
 class TranscriptionSerializer
   include FastJsonapi::ObjectSerializer
 
-  attributes :workflow_id, :group_id, :text, :status, :flagged, :updated_at, :updated_by
+  attributes :workflow_id, :group_id, :status, :flagged, :updated_at, :updated_by
+  attribute :text, if: proc { |_record, params|
+    params && params[:serialize_text] == true
```

Comment:
Can params ever be undefined here? I think even a request w/o any explicit params will still have the action in the params, so you probably don't need the first part of this clause.

Reply:
Oh huh, you're right. Good call.